mycity/mycity/intents/feedback_intent.py

```diff
@@ -46,13 +46,18 @@ def submit_feedback(mycity_request):
     else:
         feedback_type = intent_variables['FeedbackType']['value']
         feedback_text = intent_variables['Feedback']['value']
-        status = send_to_slack(
-            build_slack_message(feedback_type, feedback_text)
-        )
-        if status == 200:
-            mycity_response.output_speech = speech_constants.BIG_THANKS
-        else:
+
+        try:
+            status = send_to_slack(
+                build_slack_message(feedback_type, feedback_text)
+            )
+            if status == 200:
+                mycity_response.output_speech = speech_constants.BIG_THANKS
+            else:
+                mycity_response.output_speech = speech_constants.PROBLEM_SAVING_FEEDBACK
+        except requests.exceptions.MissingSchema:
```

Comment:
Per offline discussion, why don't we catch a general exception here.

Reply:
Done